<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;SLYR Community Edition&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;This version of the SLYR plugin has limited functionality. For full ArcMap and ArcGIS Pro conversion support, please see &lt;/span&gt;&lt;a href=&quot;https://north-road.com/slyr/&quot;&gt;&lt;span style=&quot; font-style:italic; text-decoration: underline; color:#0000ff;&quot;&gt;https://north-road.com/slyr/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Plugin Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="enable_verbose_log">
        <property name="text">
         <string>Verbose logging during conversion</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Conversion Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="convert_font_to_simple_marker">
        <property name="text">
         <string>Convert font markers to simple marker symbols where possible</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="convert_font_to_svg">
        <property name="text">
         <string>Convert font markers to SVG files</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="symbol_units"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="apply_tweaks">
        <property name="toolTip">
         <string>If checked, symbol settings will be tweaked during conversion to result in a closer visual match between ArcGIS rendering and QGIS rendering. If unchecked, symbol settings will be converted exactly, without changes.</string>
        </property>
        <property name="text">
         <string>Tweak symbol conversion for better visual match</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Units for symbols</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="replace_http_check">
        <property name="toolTip">
         <string>Replaces &quot;http&quot; references in WMS and other online service URLs with &quot;https&quot;</string>
        </property>
        <property name="text">
         <string>Replace http with https for remote services</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>SDE Database Conversion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Table name conversion</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Default primary key</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="sde_primary_key_line_edit"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="sde_table_name_conversion_combo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>MDB Tools</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_mdb_tools_win">
        <property name="text">
         <string>The MDB Tools utility can be downloaded from &lt;a href=&quot;https://github.com/lsgunth/mdbtools-win/archive/master.zip&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://github.com/lsgunth/mdbtools-win/archive/master.zip&lt;/span&gt;&lt;/a&gt;.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextBrowserInteraction</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Path to MDB tools</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsFileWidget" name="mdbtools_path_widget" native="true">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>&lt;span style=&quot; font-style:italic;&quot;&gt;The MDB tools utility is required for conversion of ESRI .style databases.&lt;/span&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Inkscape</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Path to Inkscape</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>&lt;span style=&quot; font-style:italic;&quot;&gt;An Inkscape installation is required for conversion of EMF pictures embedded in symbol definitions.&lt;/span&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsFileWidget" name="inkscape_path_widget" native="true">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>56</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>enable_verbose_log</tabstop>
  <tabstop>convert_font_to_simple_marker</tabstop>
  <tabstop>convert_font_to_svg</tabstop>
  <tabstop>apply_tweaks</tabstop>
  <tabstop>symbol_units</tabstop>
  <tabstop>replace_http_check</tabstop>
  <tabstop>sde_primary_key_line_edit</tabstop>
  <tabstop>sde_table_name_conversion_combo</tabstop>
  <tabstop>mdbtools_path_widget</tabstop>
  <tabstop>inkscape_path_widget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
